pkg/sql/sessiondata/search_path.go

```diff
@@ -211,6 +225,13 @@ func (iter *SearchPathIter) Next() (path string, ok bool) {
 			}
 			return iter.tempSchemaName, true
 		}
+		// pg_extension should be read before delving into the schema.
+		if iter.paths[iter.i-1] == "public" && iter.implicitPgExtension {
```

Comment:
Use the constant `PublicSchemaName`?

Reply:
can't - circular dependency.

Reply:
well i can move stuff around....update incoming!